content/04.network_applications.md

```diff
@@ -9,11 +9,43 @@
+In this section we discuss approaches that utilize the above methods to project knowledge graphs into a low dimensional space.
+
+SVD [@doi:10.1007/BF02288367] is an approach that can project knowledge graphs into a low dimensional space.
+The input for this technique is an adjacency matrix ($A$), which is a square matrix where rows and columns represent nodes and each entry represents the presence of an edge between two nodes. 
+This adjacency matrix ($A$) gets decomposed into three parts: a square matrix $Σ$ and a set of two small rectangular matrices $U$ and $V^{T}$.
+This values within $Σ$ are called singular values, which akin to eigenvalues [@doi:10.1007/BF02288367].
+Each row in $U$ and each column in $V^{T}$ represents nodes projected onto a low dimensional space [@doi:10.1007/BF02288367; @doi:10/bm8dnf].
+In practice $U$ is usually used to represent nodes in a knowledge graph, but $V^{T}$ can also be used [@doi:10.1007/BF02288367; @raw:Jiezhon2018].
+Overall, SVD is a universal algorithm and is typically used as a baseline to compare to other approaches [@arxiv:1906.05017].
+
+Laplacian eigenmaps is an approach that projects knowledge graphs into a low dimensional space while preserving the graph's local structure.
+Similar to SVD this approach takes an adjacency matrix ($A$) as input as well as a degree matrix ($D$): a diagonal matrix where each entry represents the number of edges connected to a node.
+The adjacency matrix is converted into a laplacian matrix ($L$) by subtracting it from the degree matrix $L=D-A$.
+Once the laplacian matrix is constructed, this algorithm uses linear algebra to calculate eigenvalues and eigenvectors from the matrix ($Lx = \lambda Dx$).
+The generated eigenvectors represent nodes projected onto a low dimensional space [@doi:10.1162/089976603321780317].
+A number of approaches have used variants of this algorithm to perform their own node projection [@raw:gongreplearn; @doi:10.1109/TKDE.2016.2606098; @arxiv:1905.09763].
+Typically eigenmaps work well when knowledge graphs have a sparse number of edges between nodes and struggle with denser networks [@arxiv:1906.05017; @doi:10.1371/journal.pcbi.1005621; @arxiv:1905.09763].
+A future direction is to adapt these methods to scale to knowledge graphs that have a large number of edges.
+
+Matrix factorization is a powerful technique that uses a matrices such as  an adjacency matrix as input.
+Common approaches involve using SVD, Laplacian eigenmaps or variants of the two to perform embeddings.
+Despite reported success, the dependence on matrices like an adjacency matrix creates an issue of scalability.
+As knowledge graphs asymptotically increases in the number of nodes and edges, these approaches would be hard to utilize.
+Furthermore, these methods treat all edge types the same, but a possible extension would be to incorporate node and edge types as sources of input.
```

Comment:
The methods like SVD, etc, or the specific approaches using them for knowledge graphs?